lib/server.js

```diff
@@ -836,6 +836,10 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
   self.getUtxos(function(err, utxos) {
     if (err) return cb(err);
 
+    utxos = _.reject(utxos, function(utxo) {
```

Comment:
If I got you right, I'd need to index ``utxosToExclude`` and iterate through ``utxos``.